tests/src/test/java/org/jboss/windup/tests/bootstrap/migrate/ExportCsvTest.java

```diff
@@ -33,5 +39,40 @@ public void exportCsv() throws IOException {
 
         File appTagsCsv = new File(tmp.getRoot(), "ApplicationFileTechnologies.csv");
         assertTrue(appTagsCsv.exists());
+
+        List<List<String>> records = readCSVFile(appTagsCsv);
+        records.forEach(record ->
+                {
+                   assertTrue(isRowSorted(record));
+                }
+        );
+    }
+
+    private List<List<String>> readCSVFile(File csvFile) throws IOException
+    {
+        List<List<String>> rows = new ArrayList<>();
+        try (BufferedReader buffer = new BufferedReader(new FileReader(csvFile))) {
+            String lineString;
+            while ((lineString = buffer.readLine()) != null) {
+                String[] columns = lineString.split(DELIMITER);
+                rows.add(Arrays.asList(columns));
+            }
+        }
+
+        return rows;
+    }
+
+    public boolean isRowSorted(List<String> row)
+    {
+        //This loop ignores the first entry in the list (ie where i=0) which is the app name,
+        //only the entries after that are sorted
+        for(int i=1;i<row.size()-1;i++)
+        {
+            if(row.get(i).compareTo(row.get(i+1))>0)
+            {
+                return false;
+            }
+        }
+        return true;
```

Comment:
Wdyt about using a java 8 method to check lists ?
`public boolean isRowSorted(List<String> info) {
    return info.equals(info.stream().sorted().collect(Collectors.toList()));
}`

Reply:
We can't use exactly this as it was written because the first item in the row is always the app name. After that they are sorted.  I've altered the readCsv method to remove the first item from each row so the sort check can work like this.

Reply:
`public boolean isRowSorted(List<String> info) {
    return info.equals(info.stream().skip(1).sorted().collect(Collectors.toList()));
}`

Reply:
But it's ok with your approach.